src/storage/src/source/kafka.rs

```diff
@@ -319,23 +318,11 @@ impl SourceRender for KafkaSourceConnection {
                 let partition_info = Arc::downgrade(&partition_info);
                 let topic = topic.clone();
                 let consumer = Arc::clone(&consumer);
-                let metadata_refresh_interval = connection
-                    .options
-                    .get("topic.metadata.refresh.interval.ms")
-                    // Safe conversion: statement::extract_config enforces that option is a value
-                    // between 0 and 3600000
-                    .map(|s| match s {
-                        StringOrSecret::String(s) => Duration::from_millis(s.parse().unwrap()),
-                        StringOrSecret::Secret(_) => unreachable!(),
-                    })
-                    // By default, rdkafka will check for updated metadata every five minutes:
-                    // https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
-                    .unwrap_or_else(|| Duration::from_secs(15));
 
                 // We want a fairly low ceiling on our polling frequency, since we rely
                 // on this heartbeat to determine the health of our Kafka connection.
-                let metadata_refresh_frequency =
-                    metadata_refresh_interval.min(Duration::from_secs(60));
+                let topic_metadata_refresh_interval =
```

Comment:
nit: this variable name confused me. Felt unclear why this is the same as the setting albeit with a ceiling applied. maybe something like `topic_metadata_refresh_polling_interval`?

Reply:
Changed to `poll_interval`!